IndexedDB/event-dispatch-active-flag.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>Transaction active flag is set during event dispatch</title>
+<link rel="help" href="https://w3c.github.io/IndexedDB/#fire-success-event">
+<link rel="help" href="https://w3c.github.io/IndexedDB/#fire-error-event">
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script src=support.js></script>
+<script>
+
+indexeddb_test(
+  (t, db, tx) => {
+    db.createObjectStore('store');
+  },
+  (t, db) => {
+    const tx = db.transaction('store');
+    const finish = keep_alive(tx, 'store');
+    assert_true(is_transaction_active(tx, 'store'),
+                'Transaction should be active after creation');
+
+    const request = tx.objectStore('store').get(0);
+    request.onerror = t.unreached_func('request should succeed');
+    request.onsuccess = () => {
+      assert_true(is_transaction_active(tx, 'store'),
+                  'Transaction should be active during success handler');
+
+      let saw_handler_promise = false;
+      Promise.resolve().then(t.step_func(() => {
+        saw_handler_promise = true;
+        assert_true(is_transaction_active(tx, 'store'),
+                    'Transaction should be active in handlers microtasks');
```

Comment:
Agreed. I'd be ok with `handler microtasks` if you don't want to change the outer quotes. I think the grammar still parses correctly, even though the form you suggested definitely has the lightest cognitive load.